docs/user_guide/1_2_setting_up_arm.md

```diff
@@ -0,0 +1,159 @@
+# Setting up the arm
+
+1. Unpack the robot arm and the control box.
+2. Mount the arm on the table. If you have a demo table, the cable socket should point roughly towards the wire hole in the table. Place the base of the robot onto the mounting plate and add screws from the top. 
+   ```eval_rst
+   .. image:: ../img/ur_position_on_table.png
+   ```
+3. Place the control box on its foot and under the table so its not trip hazard.
+   ```eval_rst
+   .. image:: ../img/ur10_control_box.jpg
+   ```
+4. Plug on the robot cable between the robot and the control box.
+   ```eval_rst
+   .. image:: ../img/ur10_control_box_cables.jpg
+   ```
+5. Plug in the mains plug of the control box.
+
+To quickly start up the robot after it has been installed, perform the following steps:
+1. Press the power button on the teach pendant.
+2. Wait a minute while the system is starting up, displaying text on the touch screen.
+3. Press ON button on the touch screen. Wait a few seconds until robot state changes to idle.
+4. Press START button on the touch screen. The robot now makes a sound and moves a little while releasing the brakes.
+
+## Configuring the network
+
+In order to use the robot with our driver you need to change the network setup of the robot via the pendant, performing the following steps:
+1. To setup the IP of the robot Press Exit on Initialization screen:
+
+   ```eval_rst
+   .. image:: ../img/configure_arm_1.jpg
+   ```
+   
+   You should see the following screen:
+   ```eval_rst
+   .. image:: ../img/configure_arm_2.jpg
+   ```
+   
+2. Press the "Setup Robot" button and you should see the following screen:
+   ```eval_rst
+   .. image:: ../img/configure_arm_3.jpg
+   ```
+   
+3. Then press the "Network" button. In this screen, you need to enable the network by clicking the "Static Address" radio button. Change the IP address and Subnet mask as shown below:
+   * IP address: 192.168.1.1
+   * Subnet mask: 255.255.255.0
+   
+   ```eval_rst
+   .. image:: ../img/configure_arm_4.jpg
+   ```
+4. Press "Apply" when you finish.
+
+## Arm Calibration Procedure
```

Comment:
The title is a bit misleading, we're not really calibrating the arm but calibrating marker position with respect to the arm